src/libNetwork/Blacklist.cpp

```diff
@@ -52,6 +52,7 @@ bool Blacklist::Exist(const uint128_t& ip, const bool strict) {
 /// Reputation Manager may use this function
 void Blacklist::Add(const uint128_t& ip, const bool strict,
                     const bool ignoreWhitelist) {
+  return;
```

Comment:
I see blacklisting is disabled by this return statement, please confirm this is deliberate.